src/test/resources/reference.conf

```diff
@@ -33,5 +33,5 @@ dataproc {
   dataprocDefaultZone = ""
   serviceAccount = ""
   pathToLeonardoPem = ""
-  clusterUrlBase = ""
+  clusterUrlBase = "http://leonardo/"
```

Comment:
Now that we construct a `URL` it needs to be valid.  I'm going to keep this commit separate.